jdaviz/app.py

```diff
@@ -955,6 +955,16 @@ def get_subsets(self, subset_name=None, spectral_only=False,
 
         all_subsets = {}
 
+        # If we only want one subset, no need to loop through them all
+        if subset_name is not None:
```

Comment:
Do we also have to guard against empty string?

`ValueError:  not in []`
```suggestion
        if subset_name not in (None, ""):
```

Reply:
I don't think so? But I guess it doesn't hurt to be thorough.

Reply:
Well, the tests are failing and suggesting that `subset_name` is empty string. 🤷‍♀️

Reply:
Ah, I didn't see the failing tests - I was impatient and didn't let the local ones get that far. I'll see if that helps.